R/download_zenodo.R

```diff
@@ -14,30 +15,32 @@
 #' @importFrom jsonlite fromJSON
 #' @importFrom tools md5sum
 #' @importFrom utils tail
-#'
+#' @importFrom assertthat
+#' assert_that
+#' is.string
 #'
 #' @export
 #' @family download
 #'
 #' @examples
 #' \dontrun{
-#' # Single zip file deposition
-#' download_zenodo(".", "10.5281/zenodo.1283345")
-#' # Multiple files deposition
-#' download_zenodo(".", "10.5281/zenodo.1172801")
-#' # Single pdf file deposition
-#' download_zenodo(".", "10.5281/zenodo.168478")
+#' # Single zip file download
```

Comment:
The examples are somewhat misleading. They download the entire repository at the DOI. The examples suggest that you can specify what to download for the repository.

Reply:
Resolved in c698835